src/pages/NotFound.jsx

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+function NotFound() {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    setTimeout(() => {
+      navigate("/", { message: "Ups, hier geht es wohl nicht weiter" });
```

Comment:
Is this message being used on the destination page? If not, remove!

Reply:
Removed :)